dockstore-client/src/main/java/io/dockstore/client/cli/nested/WorkflowClient.java

```diff
@@ -663,9 +664,29 @@ protected void handlePublishUnpublish(String entryPath, String newName, boolean
                     publish(true, entryPath);
                 }
             } else {
-                //for workflows method currently doesn't work with --entryname flag
-                errorMessage("Parameter '--entryname' not valid for workflows. See `workflow publish --help` for more information.",
-                    CLIENT_ERROR);
+                try {
```

Comment:
GitHub doesn't let me comment on lines you haven't changed, but I'm reading the whole method to better understand what is going on, and noticed this: on line 650/651 the error message is saying it can't publish/unpublish `newName`, but it doesn't even use `newName` on the previous line. I think you want that to be `entryPath`.

Reply:
Updated. Now reports that the error occurred when trying to locate the registered workflow at `entryPath`.